.env.example

```diff
@@ -0,0 +1,13 @@
+PORT = your_port_here
+
+GITHUB_SECRET = github_secret_you_enter_while_creating_webhhok
+BRANCH_NAME = main
+
+
+PROJECT_PATH = ~/Desktop/api-bench/
+DIST_PATH = ~/Desktop/api-bench-build/
```

Comment:
We have multiple projects to build, so this can't work

Reply:
`PROJECT_PATH` is like the root directory where all projects are placed and so is for the `DIST_PATH`(will be renaming it in the next commit) for Destination.